### PR TITLE
Added 05.multi-turn-prompt

### PR DESCRIPTION
**Pull request by tracyboehrer**

### CODE REVIEW

File: samples/05.multi-turn-prompt/helpers/activity_helper.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        channel_id=activity.channel_id,
+        conversation=ConversationAccount(
+            is_group=activity.conversation.is_group,
+            id=activity.conversation.id,
+            name=activity.conversation.name,
+        ),
+        text=text or "",
+        locale=locale or "",
+        attachments=[],
+        entities=[],
```
Comment (axelsrz) [inline samples/05.multi-turn-prompt/helpers/activity_helper.py]: I think you don't use the create_activity_reply method anywhere, you could remove this file

File: samples/05.multi-turn-prompt/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# Further reading
+
+- [Bot Framework Documentation](https://docs.botframework.com)
+- [Bot Basics](https://docs.microsoft.com/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0)
+- [Dialogs](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-concept-dialog?view=azure-bot-service-4.0)
+- [Gathering Input Using Prompts](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-prompts?view=azure-bot-service-4.0&tabs=csharp)
+- [Activity processing](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-concept-activity-processing?view=azure-bot-service-4.0)
+- [Azure Bot Service Introduction](https://docs.microsoft.com/azure/bot-service/bot-service-overview-introduction?view=azure-bot-service-4.0)
+- [Azure Bot Service Documentation](https://docs.microsoft.com/azure/bot-service/?view=azure-bot-service-4.0)
+- [.NET Core CLI tools](https://docs.microsoft.com/en-us/dotnet/core/tools/?tabs=netcore2x)
```
Comment (axelsrz) [inline samples/05.multi-turn-prompt/README.md]: We should remove this and add a link to the python CLI tools when available

File: samples/05.multi-turn-prompt/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- [Azure CLI](https://docs.microsoft.com/cli/azure/?view=azure-cli-latest)
+- [Azure Portal](https://portal.azure.com)
+- [Language Understanding using LUIS](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/)
```
Comment (axelsrz) [inline samples/05.multi-turn-prompt/README.md]: Not relevant to this sample

File: samples/05.multi-turn-prompt/config.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+
+""" Bot Configuration """
+
+
+class DefaultConfig(object):
```
Comment (axelsrz) [inline samples/05.multi-turn-prompt/config.py]: not necessary to inherit from object

File: samples/05.multi-turn-prompt/app.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    # NOTE: In production environment, you should consider logging this to Azure
+    #       application insights.
+    print(f"\n [on_turn_error]: { error }", file=sys.stderr)
+    # Send a message to the user
+    await context.send_activity("Oops. Something went wrong!")
+    # Clear out state
+    await CONVERSATION_STATE.delete(context)
+
+
+ADAPTER.on_turn_error = on_error
```
[the remark this answers is not in the document]
Comment (tracyboehrer) [reply samples/05.multi-turn-prompt/app.py]: It now matches the template.

File: samples/05.multi-turn-prompt/helpers/activity_helper.py (hunk 1 above)
Comment (tracyboehrer) [reply samples/05.multi-turn-prompt/helpers/activity_helper.py]: Removed

File: samples/05.multi-turn-prompt/config.py (hunk 4 above)
Comment (tracyboehrer) [reply samples/05.multi-turn-prompt/config.py]: Corrected

File: samples/05.multi-turn-prompt/README.md (hunk 3 above)
Comment (tracyboehrer) [reply samples/05.multi-turn-prompt/README.md]: For now, I removed all Azure links.  I would assume once the steps to deploy is added to the README like dotnet has, we can add back in the relevant links.

**Pull request merged**